data/campaigns/Delfadors_Memoirs/scenarios/14_Shadows.cfg

```diff
@@ -328,10 +328,19 @@
             [/not]
         [/filter_condition]
 
-        # do not allow an elvish victory before Delfador shows up
-
-        {NAMED_UNIT 2 (Draug)   42 3 Krumful _"Krumful" (facing=sw)}
-        {NAMED_UNIT 2 (Spectre) 42 2 Unhul _"Unhul" (facing=sw)}
+        # try to impede an elvish victory before Delfador shows up
+        [message]
+            speaker=Rudimil
+            message= _ "They are too close, Krumful, Unhul, arise!"
```

Comment:
As @knyghtmare suggested, I think there needs to be an additional pause in between the words 'close' and 'Krumful'. I'm not too fussed over what particular punctuation is used, and either of the suggested alternatives work for me.

Reply:
Im no linguist nor native english speaker, so I will do whatever y'all think is best with no issue.

Reply:
I'd say either a period or semicolon, your choice which.